src/app/services/object-loader/asset-loader.ts

```diff
@@ -0,0 +1,128 @@
+    new CubeMap('Medborgarplatsen', '/assets/cubemaps/urban-skyboxes/Medborgarplatsen/'),
+    new CubeMap('Roundabout', '/assets/cubemaps/urban-skyboxes/Roundabout/'),
+    new CubeMap('SaintLazarusChurch', '/assets/cubemaps/urban-skyboxes/SaintLazarusChurch/'),
+    new CubeMap('SaintLazarusChurch2', '/assets/cubemaps/urban-skyboxes/SaintLazarusChurch2/'),
+    new CubeMap('SaintLazarusChurch3', '/assets/cubemaps/urban-skyboxes/SaintLazarusChurch3/'),
+    new CubeMap('UnionSquare', '/assets/cubemaps/urban-skyboxes/UnionSquare/'),
+    new CubeMap('Bridge', '/assets/cubemaps/bridge-skyboxes/Bridge/'),
+    new CubeMap('Bridge2', '/assets/cubemaps/bridge-skyboxes/Bridge2/'),
+  ];
+
+  public readonly playerModels: Map<PlayerModel, PlayerModelData> = new Map<PlayerModel, PlayerModelData>([
+    [PlayerModel.bcap_NukaCola, new PlayerModelData('default')],
+    [PlayerModel.bcap_CocaCola, new PlayerModelData('cocaCola')],
+    [PlayerModel.bcap_Developer, new PlayerModelData('dev')],
+    [PlayerModel.bcap_Jagermeister, new PlayerModelData('jagermeister')],
+    [PlayerModel.bcap_Murica, new PlayerModelData('murica')],
+    [PlayerModel.bcap_hb, new PlayerModelData('hb')],
+    [PlayerModel.bcap_OurAnthem, new PlayerModelData('ourAnthem')],
+    [PlayerModel.bcap_Schmucker, new PlayerModelData('schmucker')],
+    [PlayerModel.bcap_Tiddies1, new PlayerModelData('kronkorken1')],
+    [PlayerModel.bcap_cat, new PlayerModelData('catGoblin')],
+    [PlayerModel.bcap_yoshi, new PlayerModelData('yoshi')],
+    [PlayerModel.bcap_niclas, new PlayerModelData('Niclas_Kronkorken')],
+    [PlayerModel.bcap_adi, new PlayerModelData('Adis_kronkorken')],
+    [PlayerModel.bcap_countcount, new PlayerModelData('countcount')],
+    [PlayerModel.bcap_gude, new PlayerModelData('gude')],
+    [PlayerModel.bcap_lordHelmchen, new PlayerModelData('lord_helmchen')],
+    [PlayerModel.bcap_skovald, new PlayerModelData('skovald', 'skovald_spec')],
+  ]);
+
+  public readonly playerModelThumbnailPath = '../assets/models/otherTex/';
+  public readonly availableDice: DiceVariations<ResourceData> = {
+    default: {
+      cname: 'diceDefault',
+      fname: 'diceDefault.glb',
+      objectCache: undefined,
+    },
+    dice: {
+      cname: 'diceModel',
+      fname: 'dice/scene.gltf',
+      objectCache: undefined,
+    },
+    dice2: {
+      cname: 'diceModel2',
+      fname: 'dice2/scene.gltf',
+      objectCache: undefined,
+    },
+  };
+
+  public readonly availableFigures: FigureVariations<ResourceData> = {
+    default: {
+      cname: 'figureDefault',
+      fname: 'figureDefault.glb',
+      objectCache: undefined,
+    },
+  };
+
+  /** Helper functions to make loader access cleaner */
+  public loadTexture(
+    path: string,
+    onLoad?: (texture: Texture) => void,
+    onProgress?: (event: ProgressEvent) => void,
+    onError?: (event: ErrorEvent) => void
+  ): Texture {
+    console.log('Loading texture: ', path);
+    const default_onLoad = (t: Texture) => {
```

Comment:
call supplied functions functions inside the default functions. Then, the default actions will also be executed when a function is supplied

Reply:
it was done